test/controllers/v1/events_controller_test.rb

```diff
@@ -0,0 +1,64 @@
+require 'test_helper'
+
+class V1::EventsControllerTest < ActionController::TestCase
+  def setup
+    @logged_user = User.create(email: 'homer@simpsons.com', password: 'any-key', auth_token: 'stupid-flanders')
+    @other_user = User.create(email: 'bart@simpsons.com', password: 'el-barto', auth_token: 'skinner-stinks')
+    @logged_event = Event.create(name: "Moe's bar", date: '2015-06-02 12:00:00', periodicity: 1, owner: @logged_user)
+    @other_event = Event.create(name: 'Treehouse', date: '2015-05-01 12:00:00', periodicity: 5, owner: @other_user)
+  end
+
+  def teardown
+    Event.delete_all
+    User.delete_all
+  end
+
+  test 'list all events which the logged user is her owner' do
+    get :index, email: @logged_user.email, auth_token: @logged_user.auth_token
+    assert_response :success
+    assert_equal assigns(:events), [@logged_event]
+  end
+
+  test 'show event when the logged user is her owner' do
+    get :show, id: @logged_event, email: @logged_user.email, auth_token: @logged_user.auth_token
+    assert_response :success
+    assert_equal assigns(:event), @logged_event
+  end
+
+  test "NOT show event when the logged user isn't her owner" do
+    get :show, id: @other_event, email: @logged_user.email, auth_token: @logged_user.auth_token
```

Comment:
SUG: as we are using `email` & `auth_token` a lot, it could be DRYed to the setup as `@auth_params`
And then using it merging the other params to it. WDYT?